ember-toucan-core/package.json

```diff
@@ -96,7 +96,7 @@
     "postcss": "^8.2.14",
     "prettier": "^2.8.3",
     "prettier-plugin-ember-template-tag": "^0.3.0",
-    "rollup": "^2.67.0",
+    "rollup": "^3.12.1",
```

Comment:
Oh! Yeah, I remember I wanted to update this also in the blueprint, but didn't look yet into what it takes (breaking changes)... seems... nothing?